app/views/ConfirmSubmission.scala.html

```diff
@@ -140,24 +140,24 @@ <h3 class="govuk-heading-s">@messages("confirm_submission.vatDetails")</h3>
 
   <dl class="govuk-summary-list">
       <div class="govuk-summary-list_row" id="box-one">
-          <dt class="govuk-summary-list__key govuk-hint">@messages("confirm_submission.boxOne")</dt>
-          <dd class="govuk-summary-list__value govuk-hint">@messages(s"confirm_submission.boxOneDescription")</dd>
-          <dd class="govuk-summary-list__actions govuk-hint">@displayMoney(viewModel.returnDetail.box1)</dd>
+          <dt class="govuk-summary-list__key vatvc-grey-paragraph-text">@messages("confirm_submission.boxOne")</dt>
+          <dd class="govuk-summary-list__value vatvc-grey-paragraph-text">@messages(s"confirm_submission.boxOneDescription")</dd>
```

Comment:
All the box title (box 1, box 2 etc) are no longer bold on this page and doesn't match staging